cfme/tests/intelligence/reports/test_menus.py

```diff
@@ -86,8 +94,10 @@ def test_shuffle_first_level(appliance, group, report_menus):
     view = navigate_to(appliance.collections.reports, "All")
     tree = view.reports.tree.read_contents()[1]
     # Select some folder that has at least 3 children
-    folders = map(lambda item: item[0],
-                  filter(lambda item: isinstance(item[1], list) and len(item[1]) >= 3, tree))
+    folders = map(
```

Comment:
Consider actually re-writing this as comprehension instead of using `lambda` (x2), `map`, and `filter`.

```
folders = {i: i[0] for i in tree if isinstance(i[1], list) and len(i[1]) >=3}